pipeline/cefilter/ce-filter.sh

```diff
@@ -29,7 +31,7 @@ test -s "${tmp}/corpus.${SRC}" || pigz -dc "${corpus_prefix}.${SRC}.gz" >"${tmp}
 echo "### Normalizing scores"
 test -s "${tmp}/scores.nrm.txt" ||
   paste "${scores}" "${tmp}/corpus.${TRG}" |
-  parallel --no-notice --pipe -k -j "${threads}" --block 50M "python pipeline/cefilter/normalize-scores.py" |
+  parallel --no-notice --pipe -k -j "${threads}" --block 50M "python normalize-scores.py" |
```

Comment:
Why is this step necessary? marian-scorer can do normalisation (run with -n). Also, using str.split() in normalize-scores.py will not properly normalize the sentence-level scores produced by marian-scorer, as that score is based on subword tokenizations.

Reply:
It was taken from https://github.com/browsermt/students/tree/master/train-student/ce-filter, so we should ask its author. I can replace it with normalization by marian.

Reply:
Likely an issue at the time of writing those scripts. Modern marian has picked up quite a few features.

Reply:
fixed